zmon-controller-app/src/main/resources/db/migration/V7__delete_update_entity_with_admin_authority.sql

```diff
@@ -0,0 +1,57 @@
+-- update to fix return of null if there are no checks at all.
+
+DROP FUNCTION IF EXISTS zzm_data.create_or_update_entity(entity_data text, teams text[], user_name text);
```

Comment:
zzm_api. is the correct schema, stored procedures called from the controller are all in the zzm_api schema.